src/foam/comics/v2/DAOBrowseControllerView.js

```diff
@@ -77,8 +77,12 @@ foam.CLASS({
   actions: [
     {
       name: 'create',
-      isAvailable: function(config$createEnabled) {
-        return config$createEnabled;
+      isAvailable: function(config) {
+        try {
+          return ! config.create || ! config.create.f ? true : config.create.f();
```

Comment:
Why is this needed?

Reply:
fixed https://github.com/foam-framework/foam2/pull/3234